calcom/cal.com
# feat: add support for apple travel time

## What does this PR do?

Adds support for Apple's [Travel Time](https://support.apple.com/en-gb/guide/calendar/icl43600/mac) feature

See [JSFiddle](https://jsfiddle.net/kv1czg05/) for example payload and parsing.

Fixes #10474
/claim #10474 

## Requirement/Documentation

- https://community.home-assistant.io/t/caldav-add-support-for-travel-time/525502
- https://github.com/nextcloud/calendar/issues/581

## Type of change


## How should this be tested?

1. Add travel time via Apple Calendar
2. Check if it's now unavailable to book via Cal

## Mandatory Tasks


## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
